bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java

```diff
@@ -243,6 +264,27 @@ public boolean matches(String type, URI uri, String realm) {
     }
 
     /**
+     * Load proxy parameters in global variables proxyHost, proxyPort, proxyUser, proxyPassword and nonProxyHosts
+     */
+    private static void prepareProxyParams() {
+        String proxySet = System.getProperty("http.proxySet");
+        if ("true".equalsIgnoreCase(proxySet)) {
+            proxyHost = System.getProperty("http.proxyHost");
+            String proxyPortString = System.getProperty("http.proxyPort");
+            if (StringUtils.isNotBlank(proxyPortString)) {
+                try {
+                    proxyPort = Integer.valueOf(proxyPortString);
+                } catch (NumberFormatException e) {
+                    logger.warn("'{}' is not a valid proxy port - using port 80 instead");
```

Comment:
Parameter missing (@lolodomo I will send you a PR with a small change -- you can have then a look at -- it will contain this fix.).